### ISSUE TITLE
Look in all projects for sample usage.

### ISSUE THREAD

**Issue by JP01** [mzero/elk-herd-project#15]
Not sure if this one is possible, but it would be nice if you could look in ALL projects for usage of a sample, not just all patterns.
This may not be possible however so feel free to close this one :D

**Comment by mzero** [comment]
Indeed, this would be great feature... at present, there isn't any quick way to do this, as the API doesn't let one scan the projects on the machine. HOWEVER, this is something that is clearly needed... and I'm keeping this open in hopes that I can work with Elektron to find a way to do this in a future update.
